tests/ntp-time-ef--vv.out

```diff
@@ -0,0 +1,28 @@
+    1  13:23:30.254335 IP (tos 0x0, ttl 64, id 62645, offset 0, flags [DF], proto UDP (17), length 360)
```

Comment:
diff `tests/ntp-time-ef--v.out` `tests/ntp-time-ef--vv.out`: only "[udp sum ok]" difference. Any need to keep the `vv` test?

Reply:
No need to keep the test. It was just for consistency with other ntp tests, but it seems they are already removed.